deb/build/debian/jenkins.init

```diff
@@ -129,7 +129,7 @@ do_start()
 # 
 get_running() 
 {
-    return `ps -U $JENKINS_USER --no-headers -f | egrep -e '(java|daemon)' | grep -c . `
+    return `ps -U $JENKINS_USER --no-headers -f | egrep -e '(java)' | grep -v defunct | grep -c . `
```

Comment:
It's primarily in Docker due to known issues with [Docker not being familiar with Blue Oyster Cult](https://www.youtube.com/watch?v=ClQcUyhoxTg), but can potentially happen *very briefly.*